libraries/lib-ffmpeg-support/FFmpegFunctions.cpp

```diff
@@ -226,6 +220,29 @@ struct FFmpegFunctions::Private final
 
       return true;
    }
+
+   std::shared_ptr<wxDynamicLibrary> LoadLibrary(const wxString& libraryName) const
+   {
+#if defined(__WXMAC__)
+      // On macOS dyld reads environment only shen application starts.
```

Comment:
"when"

Reply:
Whoa, that was stupid of me. I rely to much on IDE to highlight typos :) CLion failed me here